pages/common/go-run.md

```diff
@@ -1,9 +1,13 @@
 # go run
 
-> Compile and run Go code.
-> Does not leave a binary on the disk.
-> More information: <https://pkg.go.dev/cmd/go#hdr-Compile_packages_and_dependencies>.
+> Compile and run Go code without saving a
```

Comment:
Can we please have the description on a sperate line and More information on a separate line